### DISCUSSION TITLE
Prioritize images

### DISCUSSION THREAD

**Discussion by yggdrasil75** [d8ahazard/sd_dreambooth_extension#995] [Q&A] [answered]
I am trying to train on a few artists styles, and of course the artist has a large number of images. some of these images are of a higher quality than older images, whether this is because the artist took more time and effort for it, or simply because of the artist learning more about the tools. all of the images selected show off the artists style, but some are slightly harsher lines or worse blending or whatever. I currently duplicate the "best" images by the artist so that they have a bigger impact, but is there a more optimal way to train it? for instance: maybe naming a subfolder "x2" could say "this should be double weighted" or should I just add "high quality" to the tags for the newer stuff and "low quality" for the oldest, or most quickly drawn stuff?

**Comment by Arilziem** [comment, marked as answer]
Doubling should work, you could also move them into a folder and add it twice as a concept.

I would not add "high quality" if you want the better ones to be the default (since I assume you do not want to add "high quality" to your prompts later on), but tagging the other ones with "low quality",  "sketch" and so on should help.

Also consider dropping some of the bad ones if you have enough images overall, since I think it is preferable to not have duplicates but rather a higher ratio of bad to good.

I'm not an expert on style though, there might be something I am missing.
